docs/data/base/components/menu/MenuSimple/system/index.tsx

```diff
@@ -96,38 +100,31 @@ const StyledMenuItem = styled(MenuItem)(
   }
 
   &:hover:not(.${menuItemClasses.disabled}) {
-    background-color: ${theme.palette.mode === 'dark' ? grey[800] : grey[100]};
-    color: ${theme.palette.mode === 'dark' ? grey[300] : grey[900]};
+    background-color: ${theme.palette.mode === 'dark' ? blue[900] : blue[50]};
+    color: ${theme.palette.mode === 'dark' ? blue[100] : blue[900]};
   }
   `,
 );
 
-const TriggerButton = styled(MenuButton)(
-  ({ theme }) => `
-  font-family: IBM Plex Sans, sans-serif;
+const TriggerButton = styled(MenuButton)`
+  font-family: 'IBM Plex Sans', sans-serif;
   font-weight: 600;
   font-size: 0.875rem;
-  box-sizing: border-box;
-  border-radius: 8px;
-  padding: 8px 16px;
   line-height: 1.5;
-  background: transparent;
-  border: 1px solid ${theme.palette.mode === 'dark' ? grey[800] : grey[200]};
-  color: ${theme.palette.mode === 'dark' ? blue[300] : blue[500]};
+  background-color: ${blue[500]};
+  padding: 8px 16px;
+  border-radius: 8px;
+  color: white;
+  transition: all 150ms ease;
   cursor: pointer;
-
-  transition-property: all;
-  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
-  transition-duration: 120ms;
+  border: none;
```

Comment:
I think we could do it, but I just wonder if it's worth it. It would imply on replacing all the buttons we have on Base UI, and doing so by adding more styles (shadows, borders, gray tones and `useTheme` when necessary)—and all that to likely be replaced by the Joy UI design in the future. 

That said, if we move forward, I believe we could tackle it on a different PR, but I went ahead and designed an example:


![image](https://github.com/mui/material-ui/assets/37222944/b4a9a3df-1def-49d0-a2e5-a3120bf9ee53)
```CSS
  font-family: IBM Plex Sans, sans-serif;
  font-weight: 600;
  font-size: 0.875rem;
  line-height: 1.5;
  background-color: ${blue[500]};
  padding: 8px 16px;
  border-radius: 8px;
  color: white;
  transition: all 150ms ease;
  cursor: pointer;
  border: 1px solid ${blue[600]};
  box-shadow: 0px 2px 6px ${
    theme.palette.mode === 'dark' ? 'rgba(0, 0, 0, 0.5)' : 'rgba(0, 0, 0, 0.2)'
  }, inset 0px 1px 1px ${blue[400]}, inset 0px -1px 1px ${blue[700]} ;

  &:hover {
    background-color: ${blue[600]};
  }

  &:focus-visible {
    box-shadow: 0 3px 20px 0 rgba(61, 71, 82, 0.1), 0 0 0 5px rgba(0, 127, 255, 0.5);
    outline: none;
  }
```

Reply:
Woof, I like this so much more! 😬 I think it's fine to do it later, but we should even do it!  And on Joy, too, to be honest! 💪 It's so much better than a plain background color. If anything, I'd tone down the box-shadow just a bit more so it isn't as elevated.

Reply:
@danilo-leal would you mind taking a look at https://deploy-preview-38820--material-ui.netlify.app/base-ui/react-button/ before I tweak the buttons everywhere? 

For now, I didn't make changes on the `css` or `Tailwind` demos, just on `system`—as soon as we're confident on the design, I'll update them all.

Reply:
They're looking great! I think I'd just tone things down a slight bit ⎯ here's what I had by playing with it:

```
  font-family: IBM Plex Sans, sans-serif;
  font-weight: 600;
  font-size: 0.875rem;
  line-height: 1.5;
  background-color: ${blue[500]};
  padding: 8px 16px;
  border-radius: 8px;
  color: white;
  transition: all 150ms ease;
  cursor: pointer;
  border: 1px solid ${blue[500]};
  box-shadow: 0 2px 1px ${
    theme.palette.mode === 'dark' ? 'rgba(0, 0, 0, 0.5)' : 'rgba(45, 45, 60, 0.2)'
  }, inset 0 1.5px 1px ${blue[400]}, inset 0 -2px 1px ${blue[600]};

  &:hover {
    background-color: ${blue[600]};
  }

  &.${buttonClasses.active} {
    background-color: ${blue[700]};
    box-shadow: none;
  }

  &.${buttonClasses.focusVisible} {
    box-shadow: 0 0 0 4px ${
    theme.palette.mode === 'dark' ? blue[300] : blue[200]};
    outline: none;
  }
```

Added a few blue shades to pull those off:

```
const blue = {
  200: '#99CCFF',
  300: '#66B2FF',
  400: '#3399FF',
  500: '#007FFF',
  600: '#0072E5',
  700: '#0066CC',
};
```

Reply:
Also, just make sure the introduction box shadow is a light blue one (at least on light mode) ⎯ the one that's there at the moment clashes a bit with the gradient background.